### PR TITLE
Updated wording for people who aren't SysAdmins, fixed padding under …

### PR DESCRIPTION
**Pull request by ChristianBingman**
…the google calender (which looks ugly btw can we fix that), Updated the events with an event. + small changes

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -21,18 +21,18 @@ module.exports = {
         src: "img/logo.png",
       },
       items: [
+        { to: "/blog", label: "Events", position: "left" },
```
Comment (clee231) [inline docusaurus.config.js:24]: Styling nit: Can we be consistent with the rest of the file and have each key-value pair on a new line?

Would "Announcements" be a more appropriate name for this section? Cuz even the first post on the blog isn't talking about any given "Event".

File: blog/2021-08-21-new-site.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+slug: new-site
+title: New ACM Website!
+author: Christian Bingman
+author_title: President 2021-2022 of ACM@UIC
+author_url: https://git.christianbingman.com/ChristianBingman
+author_image_url: https://git.christianbingman.com/avatar/19c1437930a1e9db7482a581fc45f60a?size=1160
+tags: []
+---
+
+Welcome to the new ACM@UIC website! If you are looking for our old site please refer here.
```
Comment (clee231) [inline blog/2021-08-21-new-site.md:11]: Did you mean to add a link to the old site here?

File: docusaurus.config.js [hunk 3]
```diff
@@ -21,18 +21,18 @@ module.exports = {
...
+          docId: "introduction",
+          label: "About",
         },
         {
-          type: "doc",
+          label: "Who are we?",
           position: "left",
-          docId: "introduction",
-          label: "Docs",
+          href: "/about",
```
Comment (clee231) [inline docusaurus.config.js:34]: Shall we change the URL of this page to match the label?

File: docusaurus.config.js [hunk 4]
```diff
@@ -21,18 +21,18 @@ module.exports = {
...
       },
       items: [
+        { to: "/blog", label: "Events", position: "left" },
         {
-          label: "About",
+          type: "doc",
           position: "left",
-          href: "/about",
+          docId: "introduction",
+          label: "About",
```
Comment (clee231) [inline docusaurus.config.js:29]: I feel like "Docs" or "Wiki" is still a better name for this, because when you select a subsection of the docs, it's no longer an "About"-type of page.
image

**Review by clee231 — CHANGES_REQUESTED** [review]

File: src/pages/about.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
-# About
+# Who are we?
```
Comment (bmiddha) [inline src/pages/about.md:1]: I think "About ACM@UIC" or just "About" or something along the lines of "about" would work better.

File: blog/2021-08-21-new-site.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+---
+slug: new-site
+title: New ACM Website!
+author: Christian Bingman
+author_title: President 2021-2022 of ACM@UIC
+author_url: https://git.christianbingman.com/ChristianBingman
+author_image_url: https://git.christianbingman.com/avatar/19c1437930a1e9db7482a581fc45f60a?size=1160
```
Comment (bmiddha) [inline blog/2021-08-21-new-site.md:7]: change the size of image to something more appropriate. 50px maybe 100 perhaps?

File: docusaurus.config.js (hunk 1 above)
Comment (bmiddha) [reply docusaurus.config.js:24]: I agree. Blog/News/Announcements makes more sense.

File: docusaurus.config.js (hunk 4 above)
Comment (bmiddha) [reply docusaurus.config.js:29]: +1.
Docs/Wiki is better.

File: src/components/HomepageProfiles/HomepageProfiles.tsx [hunk 7]
```diff
@@ -52,8 +52,8 @@ const sysadmins = [
   },
   {
     name: "Christian Bingman",
-    image: "https://avatars.githubusercontent.com/u/42191425?s=200&v=4",
-    url: "https://github.com/ChristianBingman",
+    image: "https://git.christianbingman.com/avatar/19c1437930a1e9db7482a581fc45f60a?size=1160",
```
Comment (bmiddha) [inline src/components/HomepageProfiles/HomepageProfiles.tsx:55]: reduce image size

File: src/components/HomepageProfiles/HomepageProfiles.tsx [hunk 8]
```diff
@@ -7,8 +7,8 @@ export type HomepageOfficerProps = Record<string, never>;
 const officers = [
   {
     name: "Christian Bingman",
-    image: "https://avatars.githubusercontent.com/u/42191425?s=200&v=4",
-    url: "https://github.com/ChristianBingman",
+    image: "https://git.christianbingman.com/avatar/19c1437930a1e9db7482a581fc45f60a?size=1160",
```
Comment (bmiddha) [inline src/components/HomepageProfiles/HomepageProfiles.tsx:10]: reduce image size

**Review by bmiddha — CHANGES_REQUESTED** [review]

**Comment by clee231** [comment]
image
This URL is kinda hard to type. Should we hyphenate this URL?

**Comment by clee231** [comment]
> …the google calender (which looks ugly btw can we fix that), Updated the events with an event. + small changes
@ChristianBingman
Yeah... We're working on it. That's just the default Google styling (They are kinda due for a UI makeover to be honest...). If you recall during the call, we mentioned making a React component to just display the upcoming events. We'll most likely throw this calendar elsewhere once that is complete.

**Comment by bmiddha** [comment]
> image
> This URL is kinda hard to type. Should we hyphenate this URL?

Hyphenated urls sounds good.
I'm not a fan of the wording of "about our SIGs". It's too wordy for a navigation panel.
